src/sagas/index.js

```diff
@@ -0,0 +1,48 @@
+import { all, call, takeEvery, put } from 'redux-saga/effects';
```

Comment:
I would separate on different files these sagas to keep everything organized. Also, there's a pattern repeated on each saga, you could abstract it to avoid repeating code.